### PR TITLE
[CPU] Fix not expected No-Preprocess Exception with RGB to BGR conversion.

### PR DESCRIPTION
**Pull request by apertovs**
### Tickets:
 - 61841

### CODE REVIEW

**Review by yury-intel — APPROVED** [review]

File: inference-engine/tests/functional/plugin/shared/include/behavior/set_preprocess.hpp [hunk 1]
```diff
@@ -992,4 +992,38 @@ TEST_P(PreprocessDynamicallyInSetBlobTest, Infer) {
     }
 }
 
+TEST_P(PreprocessTest, InferWithRGB2BGRConversion) {
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/shared/include/behavior/set_preprocess.hpp:995]: I see you added parametrized test, but don't see cpu instance. Is it expected?

File: inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp [hunk 2]
```diff
@@ -308,9 +295,27 @@ InferenceEngine::Blob::Ptr MKLDNNPlugin::MKLDNNInferRequest::GetBlob(const std::
         data = _outputs[name];
         checkBlob(data, name, false);
     }
+
     if (!data) {
         IE_THROW() << "Cannot find blob with name: " << name;
     }
+
+    // check if preprocess required, but still wasn't set
+    auto preProcessedInput = std::find_if(std::begin(_networkInputs), std::end(_networkInputs),
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp:304]: Just wondering why you moved this code block to the end of the function?

File: inference-engine/tests/functional/plugin/shared/include/behavior/set_preprocess.hpp (hunk 1 above)
Comment (apertovs) [reply inference-engine/tests/functional/plugin/shared/include/behavior/set_preprocess.hpp:995]: Yes, it's expected. Instantiation is implemented here: 
`INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, PreprocessTest,
                            ::testing::Combine(
                                    ::testing::ValuesIn(netPrecisions),
                                    ::testing::Values(CommonTestUtils::DEVICE_CPU),
                                    ::testing::ValuesIn(configs)),
                            PreprocessTest::getTestCaseName);`
in inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/set_preprocess.cpp:34

File: inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp (hunk 2 above)
Comment (apertovs) [reply inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp]: Fixed

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**